### ISSUE TITLE
Support for comics

### ISSUE THREAD

**Issue by denics** [Readarr/Readarr#46]
It would be nice to add support for comics to Readarr.

[AB#1055](https://dev.azure.com/Servarr/7ab38f4e-5a57-4d70-84f4-94dd9bc5d6df/_workitems/edit/1055)

**Comment by Pfuenzle** [comment]
Yeah that would be pretty nice

**Comment by JohanG2012** [comment]
Agree, comicvine has a great API :) https://comicvine.gamespot.com/api/

**Comment by whatgoesbump** [comment]
This function is desperately needed, or someone needs to fix Mylar, because it's TERRIBLE. Can we get Mylarr?

**Comment by bakerboy448** [comment]
Closing; no plans at this time. tagging for future consideration.

https://github.com/mylar3/mylar3 is what to use in the interim

**Comment by whatgoesbump** [comment]
> Closing; no plans at this time. tagging for future consideration.
> 
> https://github.com/mylar3/mylar3 is what to use in the interim

That's what I'm using now... it's terrible!!!

**Comment by denics** [comment]
Dear all, I agree with the label "one day" but, if possible, please keep the issue open, we never know :)
